Altis_Life.Altis/core/shops/fn_chopShopMenu.sqf

```diff
@@ -6,40 +6,45 @@
     Description:
     Opens & initializes the chop shop menu.
 */
-private ["_control","_price","_nearVehicles","_chopMultiplier","_chopable","_nearUnits"];
 if (life_action_inUse) exitWith {hint localize "STR_NOTF_ActionInProc"};
 if !(playerSide isEqualTo civilian) exitWith {hint localize "STR_NOTF_notAllowed"};
+
 disableSerialization;
-_chopable = LIFE_SETTINGS(getArray,"chopShop_vehicles");
-_nearVehicles = nearestObjects [getMarkerPos (_this select 3),_chopable,25];
-_nearUnits = (nearestObjects[player,["CAManBase"],5]) arrayIntersect playableUnits;
+
+private _chopable = LIFE_SETTINGS(getArray,"chopShop_vehicles");
+private _nearVehicles = nearestObjects [getMarkerPos (_this select 3),_chopable,25];
+private _nearUnits = (nearestObjects[player,["CAManBase"],5]) arrayIntersect playableUnits;
 if (count _nearUnits > 1) exitWith {hint localize "STR_NOTF_PlayerNear"};
 
 life_chopShop = _this select 3;
 //Error check
 if (count _nearVehicles isEqualTo 0) exitWith {titleText[localize "STR_Shop_NoVehNear","PLAIN"];};
 if (!(createDialog "Chop_Shop")) exitWith {hint localize "STR_Shop_ChopShopError"};
 
-_control = CONTROL(39400,39402);
+private _control = CONTROL(39400,39402);
+private "_className";
```

Comment:
We can move these private's to become private KEYWORD (as above). This is OK even inside the forEach as there is no performance impact of doing it, as its like a 'switch'.

Reply:
Isn't this method better ? : https://community.bistudio.com/wiki/Code_Optimisation#private

Reply:
Fair enough 👍 happy with that